src/components/pages/SerialConnectPage.tsx

```diff
@@ -8,10 +8,14 @@ import Meshtastic_Logo from "@app/assets/Mesh_Logo_Black.png";
 import SerialPortOption from "@components/Onboard/SerialPortOption";
 
 import {
+  requestAutoConnectPort,
   requestAvailablePorts,
```

Comment:
Just out of curiosity, are there any limitations of which ephemeral ports we can use? Could we ever use a virtual port (like the ACM port)? Do we have to worry about port logic at all, or does that all happen under the hood?

Reply:
In theory you could point this towards any file or directory, although the autoconnection would then fail, pushing them back to the default connection screen. Not sure how that failure would log in terminal though

Reply:
In theory you could point this towards any file or directory, although the autoconnection would then fail, pushing them back to the default connection screen. Not sure how that failure would log in terminal though